app/code/Magento/Developer/Console/Command/TemplateHintsStatusCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+/**
+ * Command to show frontend template hints status
+ */
+class TemplateHintsStatusCommand extends Command
+{
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:template-hints:status';
+
+    /**
+     * Success message
+     */
+    const SUCCESS_MESSAGE = "Template hints are %status";
+
+    /**
+     * @var ScopeConfigInterface
+     */
```

Comment:
Comments are not necessary to const

Reply:
I think the third failed check is not about `const `comment but how I'm using it: first argument of translation function, as you suggested.
`$templateHintsMessage = __(self::SUCCESS_MESSAGE, ['status' => $templateHintsStatus]);`
I'm going to change this piece of code.